lib/plugins/create/templates/azure-nodejs/serverless.yml

```diff
@@ -0,0 +1,53 @@
+# Welcome to Serverless!
+#
+# This file is the main config file for your service.
+# It's very minimal at this point and uses default values.
+# You can always add more config options for more control.
+# We've included some commented out config examples here.
+# Just uncomment any of them to get that config option.
+#
+# For full config options, check the docs:
+#    docs.serverless.com
+#
+# Happy Coding!
+
+service: azure-nodejs # NOTE: update this with your service name
+
+# You can pin your service to only deploy with a specific Serverless version
+# Check out our docs for more details
+# frameworkVersion: "=X.X.X"
+
+provider:
+  name: azure
+  location: West US
+
+plugins:
+  - serverless-azure-functions
+
+# you can add packaging information here
+#package:
+#  include:
+#    - include-me.js
+#    - include-me-dir/**
+#  exclude:
+#    - exclude-me.js
+#    - exclude-me-dir/**
+
+functions:
+  hello: 
+    handler: handler.hello
+    events: 
+      - http: true
```

Comment:
Can we showcase all the events here (commented out) so that this template is not `http` specific? Like it's done here: https://github.com/serverless/serverless/blob/d0b1195cffbcdcdc65e9fa6e69e3f99103fc2643/lib/plugins/create/templates/aws-nodejs/serverless.yml#L57-L83